### ISSUE TITLE
Add support for future absences

### ISSUE THREAD

**Issue by miDeb** [miDeb/digitales_register#13]
Example data:
```json
{
	"futureAbsences": [
		{
			"id": 2109,
			"note": "English Language Competition",
			"startDate": "2022-05-02",
			"endDate": "2022-05-02",
			"startTime": 1,
			"endTime": 7,
			"justified": 4,
			"studentId": 5864,
			"reason_user": null,
			"reason_timestamp": null,
			"reason_signature": null,
			"reason": null
		}
	]
}
```

**Comment by miDeb** [comment]
Other example:
```json
 {
      "id": 2202,
      "note": null,
      "startDate": "2022-05-06",
      "endDate": "2022-05-06",
      "startTime": 8,
      "endTime": 10,
      "justified": 1,
      "studentId": 5864,
      "reason_user": 5864,
      "reason_timestamp": "2022-05-04 22:05:42",
      "reason_signature": "Michael Debertol",
      "reason": "<censored>"
    }
```

**Comment by miDeb** [comment]
Closed by 44d29576f599e69b82faddaad575ecd0f32d9118
